docs/labwc-config.5.scd

```diff
@@ -196,7 +196,7 @@ Therefore, where multiple objects of the same kind are required (for example
 
 ## WORKSPACES
 
-*<desktops><names><name>*
+*<desktops number=""><names><name>*
```

Comment:
We ought to describe here that the number attribute specifies the minimum number of workspaces.
We should also describe that number is an optional attribute.
I’d be good to also say that if number is specified, names.name is not required.